### PR TITLE
add dev discord info to home

### PR DESCRIPTION
**Pull request by pishifat**

### CODE REVIEW

File: src/pages/Home.vue [hunk 1]
```diff
@@ -20,12 +20,35 @@
...
         </div>
 
-        <div class="container card card-body py-4">
+        <div class="container card card-body py-4 my-4">
             <h4>What is the Mappers' Guild?</h4>
             <div class="text-secondary">
                 <a href="https://osu.ppy.sh/wiki/Community/Mappers_Guild" target="_blank">The Mappers' Guild</a> is a hub for beatmaps associated with <a href="https://osu.ppy.sh/beatmaps/artists" target="_blank">osu!'s Featured Artists</a>!
             </div>
             <hr />
+            <div>For the most newest info about Featured Artist mapping, check out the channels below on the <a href="https://discord.gg/ppy" target="_blank">osu!dev Discord server</a>.</div>
```
Comment (aceticke) [inline src/pages/Home.vue:29]:
```suggestion
            <div>For the latest info about Featured Artist mapping, check out the channels below on the <a href="https://discord.gg/ppy" target="_blank">osu!dev Discord server</a>.</div>
```
this gives me "the bestest free-to-win rhythm game" vibes, anyway, "most newest" is more words than necessary and newest alone should suffice, however i feel using latest is better phrasing for this

File: src/pages/Home.vue [hunk 2]
```diff
@@ -20,12 +20,35 @@
...
+            <ul>
+                <li>
+                    <code>#featured-artists</code>
+                    <ul>
+                        <li>Events related to Featured Artists. Currently, this involves creating <a href="https://discord.com/channels/188630481301012481/1094639474941636819/1094685118603481129" target="_blank">playlists for osu!(lazer) with new Featured Artist beatmaps!</a></li>
+                        <li>Quick feedback from the mapping community on your Featured Artist maps</li>
+                        <li>Discussion related to Featured Artists</li>
```
Comment (aceticke) [inline src/pages/Home.vue:36]:
```suggestion
                        <li>Discussion and events related to Featured Artists, such as creating <a href="https://discord.com/channels/188630481301012481/1094639474941636819/1094685118603481129" target="_blank">playlists for osu!(lazer) with new Featured Artist beatmaps!</a></li>
                        <li>Quick feedback from the mapping community on your Featured Artist maps</li>
```
feels weird to have these be separate points given they are similar and both use the phrase "related to featured artists". i adjusted the playlist wording to account for this

File: src/pages/Home.vue [hunk 3]
```diff
@@ -20,12 +20,35 @@
...
+                    </ul>
+                </li>
+                <li>
+                    <code>#mappers-guild</code>
+                    <ul>
+                        <li>Mappers' Guild quest activity</li>
+                        <li>Feed of newly ranked Mappers' Guild beatmaps</li>
+                        <li>Mappers' Guild member announcements</li>
```
Comment (aceticke) [inline src/pages/Home.vue:44]:
```suggestion
                        <li>Mappers' Guild member announcements</li>
                        <li>Discussion, feedback and support related to the Mappers' Guild</li>
```
feels like a nice addition

**Review by aceticke — CHANGES_REQUESTED** [review]

File: src/pages/Home.vue (hunk 1 above)
Comment (pishifat) [reply src/pages/Home.vue:29]: this was from me rewording the sentence oops

**Pull request merged**